kDriveCore/Data/Cache/AccountManager.swift

```diff
@@ -318,9 +312,8 @@ public class AccountManager: RefreshTokenDelegate, AccountManageable {
             throw DriveError.unknownToken
         }
 
-        let apiFetcher = await AccountActor.run {
```

Comment:
I just want to note that this was a mechanism to prevent having two `getApiFetcher` calls at once, as I understand it. 

Now, the account manager is thread safe, but we might be able to call multiple `getApiFetcher(for: account.userId, token: account.token)` in parallel.

It's probably fine, but maybe we should track the `updateUser` as a task and return if already running or something.

Reply:
This was indeed to make the access thread safe however this was kind of a hack. Now it should be safe because getApiFetcher only do calls on properties that use our threadsafe collections

Reply:
perfect 🤝